roles/openshift_etcd_migrate/templates/validatemigration.py.j2

```diff
@@ -0,0 +1,91 @@
+from subprocess import PIPE, Popen
```

Comment:
@sdodson 👍. Also, this may be cleaner for getting hostname in a portable way https://docs.python.org/3/library/socket.html#socket.gethostname